popde/density_estimate.py

```diff
@@ -18,9 +18,39 @@ def __init__(self, data, backend='scipy', bandwidth=1., dim_names=None):
                points of the data define each kernel position
                each row is a point, each column is a parameter.
         kwargs:
-            bandwidth : The bandwidth of the kernel used for smoothing
+           bandwidth : The bandwidth of the kernel used for smoothing
             dim_names : sequence of dimension names, e.g. ('m1', 'z', 'chi_eff')
                         values must be strings
+        Example:
+        >>> #For two dimensional case
+        >>> rndgen = np.random.RandomState(seed=3575)
+        >>> mean1, sigma1 = 3.0, .25
+        >>> mean2, sigma2 = 14.0, 1.5
+        >>> n_samples = 1000
+        >>> sample1 = rndgen.normal(mean1, sigma1, size=n_samples)
+        >>> sample2 = rndgen.normal(mean2, sigma2, size=n_samples
+        >>> sample = np.vstack((sample1, sample2)).T #shape of data (n_points, n_features)
```

Comment:
if you have to transpose the result of vstack, then vstack is not the correct function. Look for a np array function that gives the correct shape directly